app/views/jobseekers/profiles/job_preferences/_summary.html.slim

```diff
@@ -0,0 +1,58 @@
+ruby:
+  multistep = Jobseekers::JobPreferencesForm.from_record(profile.job_preferences)
+  attributes = {
+    roles: {
+      map_values: ->(value, step) { step.options[value] },
+      join_with: "</br></br>",
+    },
+    phases: {
+      map_values: ->(value, step) { step.options[value] },
+      join_with: ", ",
+    },
+    key_stages: {
+      map_values: ->(value, step) { step.options[value] },
+      join_with: ", ",
+    },
+    subjects: {
+      join_with: ", ",
+    },
+    working_patterns: {
+      map_values: ->(value, step) { step.options[value] },
+      join_with: ", ",
+    },
+    locations: {
+      if: ->(form) { form.locations.any? },
+      map_values: ->(value, _) { "#{value[:location]} (#{t('jobs.search.number_of_miles', count: value[:radius].to_i)})" },
+      join_with: "</br>",
+    },
+  }
+
+  list = govuk_summary_list do |summary_list|
```

Comment:
from here down doesnt seem like the sort of thing that should be happening in a view template but rather extracted into a helper or similar. there is too much logic constucting the summary list

Reply:
Yes, I'm planning on moving it into a component at some point. Helpers are evil